src/app/_mocked/mock-line.component.ts

```diff
@@ -0,0 +1,127 @@
+import { Component, Input } from '@angular/core';
```

Comment:
Do we still need mock data?
If not then it is better to remove them

Reply:
Everything under `_mocked` is used to mock components so that their containing components can be unit tested.